airbyte-integrations/connectors/destination-clickhouse-v2/src/main/kotlin/io/airbyte/integrations/destination/clickhouse_v2/config/ClickhouseBeanFactory.kt

```diff
@@ -31,6 +32,11 @@ class ClickhouseBeanFactory {
                 .setPassword(config.password)
                 .setDefaultDatabase(config.resolvedDatabase)
                 .compressClientRequest(true)
+                // allow experimental JSON type
+                .serverSetting("allow_experimental_json_type", "1")
+                // allow JSON transcoding as a string
```

Comment:
nit: add something about _why_ this is needed, rather than just saying _what_ it's doing